sunpy/instr/aia/response.py

```diff
@@ -535,51 +578,93 @@ def calculate_temperature_response(self, channel):
-        #     contribution_function = self.get_contribution_function(channel, wavelength_range, specific_ion)
-        #     gofnt_array.append(contribution_function)
-        # temp_response = gofnt_array * wavelength_response
-        # print('temp response: ', self.temperature_response)
-
-        self.temperature_response =  {'temperature': temperatures, 'temperature response': temp_response}
+        # # test with wavelength response as full array?
+        # self.calculate_wavelength_response(channel, total_range=True, use_genx_values=True)
+
+        # test with wavelength response as float value for channel? -
+        self.calculate_wavelength_response(channel,  use_genx_values=True)
+
+        # # test case: one ion object - fe14
+        # fe14 = ch.ion('fe_14', temperature=t, eDensity=1.e+9, em=1.e+27)
+
+        # looping through elements and ions:
+        ion_array = []
+
+        # test case: store iron ion names
+        # for level in range(1, 15, 1):
+        #     ion_array.append('fe' + '_' + str(level))
+
+        # # test case: store ion names of elements with high solar abundance - basically same result as running all ions
+        # # (based on logarithmic abundance chart, higher than ~ 7 dex)
+        # solar_elements = ['h', 'he', 'c', 'n', 'o', 'ne', 'si', 's', 'ar', 'ca', 'ni', 'fe']
+        # for element in solar_elements:
+        #     for level in range(1, 18, 1): # max 38
+        #         ion_array.append(element + '_' + str(level))
+
+        # # test case: store every element in chianti ion names - takes a REALLY long time...
+        # for element in c.El:
+        #     for level in range(1, 38, 1):
+        #         ion_array.append(element + '_' + str(level))
+
+        # store every ion from Felding and Widing (1993) abundances: bibcode 1995ApJ...443..416L
+        ion_array = ['o_5', 'o_6', 'ne_8', 'mg_7', 'mg_10', 'si_6', 'si_7', 'si_8', 'si_9', 'si_10', 's_8', 's_9',
+                     's_10', 's_11', 's_12', 's_13', 'fe_8', 'fe_9', 'fe_10', 'fe_11', 'fe_12', 'fe_13', 'fe_14',
+                     'fe_15', 'ni_11', 'ni_12', 'ni_13', 'ni_17', 'ni_18']
+
+        # find contribution function for all ions
+        gofnt_array = []
+        for i in ion_array:
+            try:
+                ion_object = ch.ion(i, temperature=t, eDensity=1.e+9, em=1.e+27)
+                self.calculate_contribution_function(channel, wavelength_range, ion_object, chianti_Gofnt=True)
+                gofnt_array.append(self.contribution_function)
+                print(i)
+            except (AttributeError, KeyError, UnboundLocalError, IndexError):
+                # doesn't save ions if no lines found in selected interval
+                #                      missing information (Elvlc file missing)
+                #                      cause errors in ChiantyPy (Zion2Filename, self.Ip)
+                # print(i, ' contribution not calculated')
+                pass
```

Comment:
In general, `except... pass` is not a great idea because unexpected errors can sometimes sneak through unnoticed. I'd recommend raising a warning here so the user knows what's going on.